composer.json

```
@@ -37,6 +37,9 @@
         "symfony/uid": "^5.4|^6.0",
         "symfony/validator": "^5.4|^6.0"
     },
+    "suggest": {
+        "symfony/mercure": "^0.6.3"
```

```suggestion
        "symfony/mercure": "To enable live updates support"
```

(https://getcomposer.org/doc/04-schema.md#suggest)